### PR TITLE
Update Dai network address

### PR DESCRIPTION
**Pull request by chrismaree**
This PR corrects the Dai address in the `PublicNetworks.js` file. Additionally the `wethAddress`s on all networks were updated to contain the correct cammel case checksum.

### CODE REVIEW

File: common/PublicNetworks.js [hunk 1]
```diff
@@ -3,28 +3,28 @@ module.exports = {
     name: "mainnet",
     ethFaucet: null,
     etherscan: "https://etherscan.io/",
-    daiAddress: "0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359",
-    wethAddress: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"
+    daiAddress: "0x6B175474E89094C44Da98b954EedeAC495271d0F",
```
Comment (chrismaree) [inline common/PublicNetworks.js:6]: This changed from [Sai](https://etherscan.io/address/0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359) -> [Dai](https://etherscan.io/address/0x6B175474E89094C44Da98b954EedeAC495271d0F)

File: common/PublicNetworks.js [hunk 2]
```diff
@@ -3,28 +3,28 @@ module.exports = {
...
+    wethAddress: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
```
Comment (chrismaree) [inline common/PublicNetworks.js:7]: Address has not changed; just added the correct checksum to this.

Comment (nicholaspai) [reply common/PublicNetworks.js:7]: Cool. Are the other addresses checksum addresses as well?

Comment (chrismaree) [reply common/PublicNetworks.js:7]: yup

**Pull request merged**